mobify/platform-scaffold
# AMP-45: Main Menu

**JIRA**: https://mobify.atlassian.net/browse/AMP-45

## Changes
- `<LinkTile>` component 
- `<SkipLinks>` component 
- `<Button>` component 
- `<Nav>` component 
- `<NavItem>` component 
- `<NavMenu>` component 
- import components that use in App container 
- Fix App markup to make it similar as PWA App container
- Initial Navigation container 
- Project components theme 

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions AMP](http://localhost:3000/)
- Open Nav Sheet and check Link Tiles
